engine/src/main/java/com/redhat/cloud/notifications/processors/camel/CamelTypeProcessor.java

```diff
@@ -120,8 +136,13 @@ public NotificationHistory callCamel(Notification item, UUID historyId, JsonObje
         String accountId = item.getEndpoint().getAccountId();
         // the next could give a CCE, but we only come here when it is a camel endpoint anyway
         String subType = item.getEndpoint().getSubType();
+        CamelProperties camelProperties = (CamelProperties) item.getEndpoint().getProperties();
 
-        reallyCallCamel(payload, historyId, accountId, subType);
+        if (camelProperties.getSubType().equals("slack")) { // OpenBridge
```

Comment:
`camelProperties.getSubType()` is deprecated. Should this and all other occurrences in this PR be changed?